Tests/PlotTests/HTMLTests.swift

```diff
@@ -128,6 +128,55 @@ final class HTMLTests: XCTestCase {
         """)
     }
 
+    func testLinkHrefLangEnglish() {
+        let html = HTML(.head(.link(.rel(.alternate), .href("http://site/"), .hreflang(.english))))
+        assertEqualHTMLContent(html, """
+        <head><link rel="alternate" href="http://site/" hreflang="en"/></head>
+        """)
+    }
+
+    func testLinkHrefLangEnglishUS() {
+        let html = HTML(.head(.link(.rel(.alternate), .href("http://site/"), .hreflang(.english,.us))))
```

Comment:
Here for example you can use `.hreflang(.usEnglish)` to make this test pass, without introducing the `Country` enum.

Reply:
done.